llvm/test/Transforms/LICM/preversing-debugloc-fmul.ll

```diff
@@ -0,0 +1,78 @@
+; RUN: opt -passes=licm -verify-memoryssa -S < %s | FileCheck %s
+
+; Test that LICM's hoistRegin() 
```

Comment:
Is this comment unfinished?

Reply:
Oh, that must be a commit mistake. I'll fix it.